plugin/src/main/java/com/denizenscript/denizen/events/player/PlayerArmorStandManipulateScriptEvent.java

```diff
@@ -0,0 +1,100 @@
+package com.denizenscript.denizen.events.player;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.nms.NMSHandler;
+import com.denizenscript.denizen.nms.NMSVersion;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.ItemTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerArmorStandManipulateEvent;
+
+public class PlayerArmorStandManipulateScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // player edits armor stand
+    //
+    // @Group Player
+    //
+    // @Location true
+    //
+    // @Cancellable true
+    //
+    // @Triggers when a player manipulates an armor stand entity.
+    //
+    // @Switch armor_stand_item:<item> to only process the event if item held by the armor stand matches the specified item matcher.
+    // @Switch hand:<hand> to only process the event if the player is using a specific hand to interact with the armor stand. Available only on MC versions 1.19+.
+    // @Switch player_item:<item> to only process the event if the item held by the player matches the specified item matcher.
+    // @Switch slot:<slot> to only process the event if the armor stand's item slot that was interacted with is the specified slot.
+    //
+    // @Context
+    // <context.armor_stand_item> returns the ItemTag held by the armor stand.

```

Comment:
Should verify what this item actually is and update meta accordingly, see discussion on [Discord](https://discord.com/channels/315163488085475337/1011496047811506227/1220418650151059538)